compiler/noirc_errors/src/debug_info.rs

```diff
@@ -99,7 +100,8 @@ pub struct DebugInfo {
     #[serde_as(as = "BTreeMap<DisplayFromStr, _>")]
     pub locations: BTreeMap<OpcodeLocation, Vec<Location>>,
     #[serde_as(as = "BTreeMap<_, BTreeMap<DisplayFromStr, _>>")]
```

Comment:
This should be no longer needed afaik.

Reply:
Yeah I believe you are correct. I have removed it.